contrib_bots/lib/CommuteBot/CommuteBot.py

```diff
@@ -0,0 +1,150 @@
+            It looks for messages starting with '@commute'.
+
+            Users should input an origin, destination and a mode of transport.
+            '''
+
+    def triage_message(self, message):
+        # return True iff we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups!
+        if message['display_recipient'] == 'commute':
+            return False
+        is_commute = (original_content.startswith('@commute') or
+                        original_content.startswith('@commute'))
+
+        return is_commute
+
+    # allows bot to send info via private message or same stream depending on
+    # user input
+    def send_info(self, message, letter, client):
+        if message['type'] == 'private':
+            client.send_message(dict(
+                type='private',
+                to=message['sender_email'],
+                content=letter,
+            ))
+        else:
+            client.send_message(dict(
+                type='stream',
+                subject=message['subject'],
+                to=message['display_recipient'],
+                content=letter,
+            ))
+
+    def handle_message(self, message, client, state_handler):
+        original_content = message['content']
+        content_list = original_content.split()
+
+        if "help" in content_list:
+            help_info = '''
+    Obligatory Inputs:
+        Origin (origin=New York,NY,USA)
+        Destination (destinations=Chicago,IL,USA)
+        Mode Of Transport (Accepted inputs:driving, walking, bicycling, transit) (mode=driving)
+    Optional Inputs:
+        Units (Metric or imperial) (units=metric)
+        Restrictions (Accepted inputs:tolls, highways, ferries, indoor) (avoid=tolls)
+        Departure Time (Seconds After Midnight January 1, 1970 UTC) (departure_time=now)
+        Arrival Time (Seconds After Midnight January 1, 1970 UTC) (arrival_time=732283981023)
+        Language (Available languages available here: https://developers.google.com/maps/faq#languagesupport) (lan=fr)
+
+    Sample request:
+        @commute origin=Chicago,IL,USA destination=New+York,NY,USA mode=driving language=fr
+
+    Please note:
+        Fare information can be derived, though is solely dependent on the
+         availability of the information released by public transport operators
+        Duration in traffic can only be derived if a departure time is set.
+        If a location has spaces in its name, please use a + symbol in the place of the space/s
+        A departure time and a arrival time can not be inputted at the same time
+        To add more than 1 input for a category,
+        eg. more than 1 destinations, use |, eg. destinations=Trump+Tower|The+White+House
+        No spaces within addresses.
+                            '''
```

Comment:
Indentation for this code block is not in line with the variable it's stored in.

Also, please make sure your lines are 80 characters long.